### PR TITLE
Basic Details View Controller functionality, contacts not added yet

### PR DESCRIPTION
**Pull request by varunadiga**
screen shot 2019-03-07 at 12 06 11 pm

### CODE REVIEW

**Review by tirtow — COMMENTED** [review]
Not approved. Removes core feature.

File: RepresentsMe/View Controllers/DetailsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import UIKit
+
+class DetailsViewController: UIViewController {
+    @IBOutlet weak var officialName: UILabel!
+    @IBOutlet weak var officialSeat: UILabel!
+    @IBOutlet weak var officialParty: UILabel!
+    @IBOutlet weak var officialContact: UILabel!
+    @IBOutlet weak var officialPicture: UIImageView!
+    
+    var passedName = String()
```
Comment (bzsinger) [inline RepresentsMe/View Controllers/DetailsViewController.swift]:
```
var official:Official {
	didSet {
	   officialName.text = ...
	}
}
```

**Review by bzsinger — CHANGES_REQUESTED** [review]
🦇

**Review by bzsinger — APPROVED** [review]
LGTM! 🔥

**Pull request merged**